### ISSUE TITLE
Code Table Request - ear length

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6307]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

On behalf of @jldunnum @campmlc and the RANGES team, and in preparation for https://github.com/ArctosDB/arctos/issues/6111:

Merge

* ear from crown [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#ear_from_crown)
* ear from notch [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#ear_from_notch)

into one new 'ear' attribute

_**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

ear length

_**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

Length of the ear.

_**Attribute data type** If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

number+units

_**Attribute units** If numberical values should be accompanied by units, provide a link to the appropriate units table._

ctlength_units

_**Context**: Describe why this new value is necessary and existing values are not._

We have method embedded in a term (and much of it may be guesses).

_**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type

_**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

carry over from existing

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

? high ?

_**Example Data**: Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

I can work with collections to determine an appropriate migration pathway if this is approved in principle.

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by cjconroy** [comment]
If the method is "from notch" or "crown", will those values be free text in method, or will there be a dropdown to avoid any misspellings?

**Comment by dustymc** [comment]
Method is free-text. (The idea that perhaps that need some kind of expansion - eg a way to formally reference a publication, or here perhaps a categorical "supermethod" - keeps coming up, but I don't know of anything concrete on that front.)

**Comment by Jegelewicz** [comment]
ear seems like a part name? ear length?

**Comment by jldunnum** [comment]
Yea agree, ear length and hind foot length for the other

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mkoo** [comment]
Same as #6308 -- seems clear cut

Proposed New Value:
 ear length
(with definition above)

Proposed Action:
Merge ear from crown [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#ear_from_crown)
  and  ear from notch [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#ear_from_notch)

@jldunnum said yes!

**Comment by Jegelewicz** [comment]
@dustymc let me know if you want me to create the new attribute or do anything else today.

**Comment by bryansmclean** [comment]
This seems good to me (speaking on behalf of RANGES).

**Comment by ebraker** [comment]
I'm a bit worried about losing precision so I would advocate for a ear length methods code table that includes "from notch" and "from crown" as I'm certain a goodly amount of our volunteers and students will forget to record "from notch" as free text when entering records. There's still Remarks to accommodate pubs and other methods, plus the potential to expand the code table should we start measuring ears in novel ways, or need to add "other" and punt specifics to the Remarks field.

**Comment by dustymc** [comment]
I opened another issue for possible method change discussion.

I can prefix method with the current attribute type (or whatever), which I'm sure will be sufficient to update from if that discussion comes to some actionable state.

**Comment by Jegelewicz** [comment]
> I'm a bit worried about losing precision

Might be mitigated by #6510

**Comment by dustymc** [comment]
I'm not sure what to do here - boxes are checked, but there's what looks like an objection. @ArctosDB/arctos-working-group-officers advice please.

**Comment by Jegelewicz** [comment]
@ebraker can we go ahead?

**Comment by dustymc** [comment]
@adhornsby to download users must go through...

<img width="556" alt="Screenshot 2023-08-14 at 12 55 44" src="https://github.com/ArctosDB/arctos/assets/5720791/e4ad1fca-b4be-4c9f-83b9-85a3e08a6619">

Would adding a "these aren't the full data...." disclaimer to that alleviate any concerns?

**Comment by Jegelewicz** [comment]
These are summary data, for more complex information that includes all possible data, .....

**Comment by Jegelewicz** [comment]
If we merge, then method needs to be appended to the value and units in flat.

But nobody wants to merge. These are two different things. At some point we will need ear length?
